lib/kitchen/driver/ec2.rb

```diff
@@ -536,12 +528,19 @@ def wait_until_ready(server, state)
             state[:hostname] = hostname(aws_instance, nil)
           end
           if ready && windows_os?
-            output = server.console_output.output
-            unless output.nil?
-              output = Base64.decode64(output)
-              debug "Console output: --- \n#{output}"
+            if instance.transport[:username] =~ /administrator/i &&
+                instance.transport[:password].nil?
+              # If we're logging into the administrator user and a password isn't
```

Comment:
Should we put in a debug entry on this?

Reply:
So I had thought about that, but `fetch_windows_admin_password` already logs out so the std output ends up with this:

```
EC2 instance <i-0fc07ef6312345678> created.
Waited 0/600s for instance <i-0fc07ef6312345678> to become ready.
Waited 10/600s for instance <i-0fc07ef6312345678> to become ready.
Waited 20/600s for instance <i-0fc07ef6312345678> to become ready.
Waited 0/600s for instance <i-0fc07ef6312345678> to fetch windows admin password.
Waited 10/600s for instance <i-0fc07ef6312345678> to fetch windows admin password.
Waited 20/600s for instance <i-0fc07ef6312345678> to fetch windows admin password.
Retrieved Windows password for instance <i-0fc07ef6312345678>.
```

If you still think there's value in the debug statement, I can add one.

Reply:
Nope lgtm.As long as there are some useful logs that's all that matters to me